configs/config.json

```diff
@@ -1,14 +1 @@
-{
-  "pth_path": "weights/kikiV1.pth",
-  "index_path": "logs/kikiV1.index",
-  "sg_input_device": "VoiceMeeter Output (VB-Audio Vo (MME)",
-  "sg_output_device": "VoiceMeeter Aux Input (VB-Audio (MME)",
-  "threhold": -45.0,
-  "pitch": 0.0,
-  "index_rate": 1.0,
-  "block_time": 0.09,
-  "crossfade_length": 0.15,
-  "extra_time": 5.0,
-  "n_cpu": 8.0,
-  "f0method": "rmvpe"
-}
+{"pth_path": "F:/src/Retrieval-based-Voice-Conversion-WebUI/assets/weights/Mahiro.pth", "index_path": "F:/src/Retrieval-based-Voice-Conversion-WebUI/logs/Mahiro_added_IVF94_Flat_nprobe_3.index", "sg_input_device": "Microphone (USB Advanced Audio Device) (Windows DirectSound)", "sg_output_device": "Output 1/2 (Komplete Audio 6 MK2) (Windows DirectSound)", "threhold": -45.0, "pitch": 0.0, "index_rate": 1.0, "block_time": 0.09, "crossfade_length": 0.15, "extra_time": 5.0, "n_cpu": 8.0, "f0method": "rmvpe"}
```

Comment:
You'd better keep a human-readable ident.

Reply:
Sorry, this was a mistake